spec/services/solidus_pay_tomorrow/client/create_order_service_spec.rb

```diff
@@ -39,9 +39,9 @@ def expected_body
         zip: order.bill_address.zipcode,
         state: order.bill_address.state.abbr,
         email: order.email,
-        returnUrl: 'https://domain.com/webhooks/return',
-        cancelUrl: 'https://domain.com/webhooks/cancel',
-        notifyUrl: 'https://domain.com/webhooks/notify',
+        returnUrl: "#{SolidusPayTomorrow.config.base_url}/return",
+        cancelUrl: "#{SolidusPayTomorrow.config.base_url}/cancel",
+        notifyUrl: "#{SolidusPayTomorrow.config.base_url}/notify",
```

Comment:
Considering that `return` and `cancel` have been implemented, what do you think about using the route helpers `pay_tomorrow_return_path` and `pay_tomorrow_cancel_path`?

(They need to be scoped under Spree::Core::Engine.routes.url_helpers )

Reply:
The problem with this is ->
`pay_tomorrow_return_path` = `/pay_tomorrow/return`
&
`SolidusPayTomorrow.config.base_url` ~ `https://80a5-125-63-97-62.in.ngrok.io/pay_tomorrow`

Hence if we'll try to construct returnUrl,
"#{SolidusPayTomorrow.config.base_url}#{spree.pay_tomorrow_return_path}"

it'll be 
`https://80a5-125-63-97-62.in.ngrok.io/pay_tomorrow/pay_tomorrow/return`

The path would be duplicated so we'll have to do some hackish business in order to fix it.
Doable with URI.parse but not sure if it is worth it.

Wdyt?

Reply:
The value of `SolidusPayTomorrow.config.base_url` is decided by us. 
We can change it to `https://80a5-125-63-97-62.in.ngrok.io/` without any issue.

Reply:
Cool then, we'll change the ENV value and use `#{SolidusPayTomorrow.config.base_url}#{spree.pay_tomorrow_return_path}`

Reply:
Added this in https://github.com/nebulab/solidus_pay_tomorrow/pull/11/commits/cd13268ce6ff1f156e33617bbb8ec22de3a85167